plugins/woocommerce/package.json

```diff
@@ -113,6 +113,7 @@
 		"ie 9"
 	],
 	"dependencies": {
+		"@playwright/test": "^1.33.0",
```

Comment:
There is nothing wrong about this, just taking the opportunity to learn something new 😄:

What criteria do we follow to decide if a package should be a dependency or a devDependency?

Reply:
Actually, just an oversight on my part.  I think this actually should be a devDependency because it's not used in the running of the application in production, it's only used during development.